napari/layers/_tests/test_layer_actions.py

```diff
@@ -288,6 +288,12 @@ def test_convert_layer(layer, type_):
         )  # check array data not copied unnecessarily
 
 
+def test_convert_warns_with_projecton_mode():
+    ll = LayerList([Image(np.random.rand(10, 10), projection_mode='mean')])
+    with pytest.warns(UserWarning, match='projection mode'):
+        _convert(ll, 'labels')
```

Comment:
```suggestion
        _convert(ll, 'labels')
    assert isinstance(ll["image"], Labels)
```

Maybe also check if conversion is done?